Makefile

```diff
@@ -13,12 +13,14 @@ OPEN_URL=xdg-open
 MIGRATOR=$(PENV_BIN_PATH)alembic -c config/alembic.ini
 BWR=bower
 NPM=npm
+MJML=mjml
```

Comment:
By the way I literally have no idea why have both you and @webknjaz created these aliases.
Can anyone explain that to me, please?